### PR TITLE
Add First Person Camera Object

### PR DESCRIPTION
**Pull request by DoobesURU**
Adds support for first person camera objects such as the baron's office desk. Disappears the avatar as well.

### CODE REVIEW

File: korman/ui/ui_camera.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def draw_camera_poa_props(layout, cam_type, props):
     col.prop(trans, "poa_deceleration", text="Deceleration")
     col.prop(trans, "poa_velocity", text="Maximum Velocity")
     col = col.column()
-    col.active = cam_type == "follow"
+    col.active = cam_type == "follow" or cam_type == "firstperson"
```
Comment (Hoikas) [inline korman/ui/ui_camera.py:95]:
```suggestion
    col.active = cam_type in {"firstperson", "follow"}
```

File: korman/ui/ui_camera.py [hunk 2]
```diff
@@ -118,12 +118,12 @@ def draw_camera_pos_props(layout, cam_type, props):
     _draw_alert_prop(col, trans, "pos_velocity", cam_type,
                      alert_cam="rail", max=10.0, text="Maximum Velocity")
     col = col.column()
-    col.active = cam_type == "follow"
+    col.active = cam_type == "follow" or cam_type == "firstperson"
```
Comment (Hoikas) [inline korman/ui/ui_camera.py:121]:
```suggestion
    col.active = cam_type in {"firstperson", "follow"}
```

File: korman/ui/ui_camera.py [hunk 3]
```diff
@@ -118,12 +118,12 @@ def draw_camera_pos_props(layout, cam_type, props):
...
     col.prop(trans, "pos_cut", text="Cut Animation")
 
     # Position Offsets
     col = split.column()
-    col.active = cam_type == "follow"
+    col.active = cam_type == "follow" or cam_type == "firstperson"
```
Comment (Hoikas) [inline korman/ui/ui_camera.py:126]:
```suggestion
    col.active = cam_type in {"firstperson", "follow"}
```

**Review by Hoikas — COMMENTED** [review]
Just to make sure, this greys out everything that isn't used by first person cameras? I don't have the reference for those up atm.

**Comment by DoobesURU** [comment]
> 
> 
> Just to make sure, this greys out everything that isn't used by first person cameras? I don't have the reference for those up atm.

Sorry for the late response, but just noticed this. Yes, I believe so. A FP cam, AFAICT, seems to have pretty much the same properties as a follow cam, which is why they have shared instances in the modified UI script.

**Review by Hoikas — APPROVED** [review]

**Pull request merged**